CLI: Improve experience when running `qmk setup` on FreeBSD. (#8798)
qmk/qmk_firmware: docs/newbs_getting_started.md

[Download the latest release here.](https://github.com/qmk/qmk_toolbox/releases/latest)

> -   For Windows: `qmk_toolbox.exe` (portable) or `qmk_toolbox_install.exe` (installer)
> -   For macOS: `QMK.Toolbox.app.zip` (portable) or `QMK.Toolbox.pkg` (installer)

Reviewer: ```suggestion
* For Windows: `qmk_toolbox.exe` (portable) or `qmk_toolbox_install.exe` (installer)
* For macOS: `QMK.Toolbox.app.zip` (portable) or `QMK.Toolbox.pkg` (installer)
```

You will need to install MSYS2, Git, and the QMK CLI.

> -   Follow the installation instructions on the [MSYS2 homepage](http://www.msys2.org).
> -   Close any open MSYS2 terminals and open a new MSYS2 MinGW 64-bit terminal. NOTE: This is **not** the same as the MSYS terminal that opens when installation is completed.

Reviewer: ```suggestion
* Follow the installation instructions on the [MSYS2 homepage](http://www.msys2.org).
* Close any open MSYS2 terminals and open a new MSYS2 MinGW 64-bit terminal. NOTE: This is **not** the same as the MSYS terminal that opens when installation is completed.
```

You will need to install Git and Python. It's very likely that you already have both, but if not, one of the following commands should install them:

> -   Debian / Ubuntu / Devuan: `sudo apt install git python3 python3-pip`
> -   Fedora / Red Hat / CentOS: `sudo yum install git python3 python3-pip`
> -   Arch / Manjaro: `sudo pacman -S git python python-pip python-setuptools libffi`

Reviewer: ```suggestion
* Debian / Ubuntu / Devuan: `sudo apt install git python3 python3-pip`
* Fedora / Red Hat / CentOS: `sudo yum install git python3 python3-pip`
* Arch / Manjaro: `sudo pacman -S git python python-pip python-setuptools libffi`
```

* Fedora / Red Hat / CentOS: `yum install git python3 && python3 -m pip install qmk`
* Arch: `yay -S qmk` (or use any other AUR Helper)

> ### FreeBSD

Reviewer 2: Adding id tags helps people doing translations, and is something we've started to do recently.

```suggestion
### FreeBSD :id=freebsd
```

?> If you already know [how to use GitHub](getting_started_github.md), we recommend that you create your own fork and use `qmk setup <github_username>` to clone your personal fork. If you don't know what that means you can safely ignore this message.

> ### Note on FreeBSD Permissions

Reviewer 2: ```suggestion
### Note on FreeBSD Permissions :id=note-on-freebsd-permissions
```

It is suggested to run `qmk setup` as a non-`root` user to start with, but this will likely identify that packages need to be isntalled to your

> base system using `pkg`. Although you maybe be prompted to install dependencies, this will likely fail due to running as an unprivileged user. To

Reviewer 3: ```suggestion
base system using `pkg`. However the installation will probably fail when run as an unprivileged user. To
```
